test/inductor/test_torchinductor.py

```diff
@@ -787,6 +787,66 @@ def repeat(x, n):
         self.assertEqual(expect, actual)
         self.assertEqual(actual, repeat(x, 3))
 
+    @skipIfRocm
+    def test_neg_index(self):
+        def test(fn, inps, has_assert: bool, has_wrapping=True):
+            for dynamic in (True, False):
+                fn_opt = torch.compile(dynamic=dynamic)(fn)
+                if self.device == "cpu":
+                    code = run_and_get_cpp_code(fn_opt, *inps)
```

Comment:
Add the equivalent asserts in CPU please.

Reply:
@guilhermeleobas I saw you are adding assertions in a separate PR https://github.com/pytorch/pytorch/pull/108910. Are you going to add the checkers there too?

Reply:
Yes